haechan45th/src/routes/postRouter.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const postController = require('../controllers/postController');
+
+const router = express.Router();
+
+router.get('/', postController.getAllPost);
+router.get('/:userId', postController.getSpecificUserPost);
+router.post('/:userId/create', postController.createUserPost);
```

Comment:
```suggestion
router.post('/:userId', postController.createUserPost);
```
http request method로 구분되기 때문에 엔드포인트에 동사(액션)가 들어가지 않아도 됩니다.